components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/constants/RoleMgtConstants.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.role.v2.mgt.core.constants;
+
+/**
+ * Constants for role management.
+ */
+public class RoleMgtConstants {
+
+    public static final String USER_ID = "userId";
+    public static final String USER_IDS = "userIds";
+    public static final String USER_GROUPS = "userGroups";
+    public static final String MAIN_ROLE_UUID = "MainRoleUUID";
+    public static final String SHARED_ROLE_UUID = "SharedRoleUUID";
+
+    public static final String COMMA = ",";
+    public static final String QUESTION_MARK = "?";
+    public static final String CLOSE_PARENTHESIS = ")";
+    public static final String WHITE_SPACE = " ";
+
+    public static final String SHARING_ERROR_PREFIX = "RMT-";
+
+    /**
+     * Error messages for organization user sharing management related errors.
+     */
```

Comment:
This comment should be updated according role mgt.

Reply:
Addressed and resolved in commit: [42705b9](https://github.com/wso2/carbon-identity-framework/pull/6162/commits/42705b984942a14ec734891444d31c9d33397a6f)